optaweb-vehicle-routing-backend/src/test/java/org/optaweb/vehiclerouting/plugin/planner/SolverIntegrationTest.java

```diff
@@ -103,8 +106,9 @@ void removing_visits_should_not_fail() {
                 singletonList(location(2))
         );
 
-        sf.getSolverConfig().setDaemon(true);
-        Solver<VehicleRoutingSolution> solver = sf.buildSolver();
+        solverConfig.setDaemon(true);
```

Comment:
Why not setDaemon(true) in the setup and directly build the SolverFactory there?
both tests set it to true